code/modules/projectiles/gun_helpers.dm

```diff
@@ -290,6 +290,9 @@ should be alright.
 /obj/item/weapon/gun/proc/is_wielded() //temporary proc until we get traits going
 	return CHECK_BITFIELD(flags_item, WIELDED)
 
+/obj/item/weapon/gun/proc/is_deployed()
+	return CHECK_BITFIELD(flags_item, IS_DEPLOYED)
```

Comment:
kill this proc, replace every is_deployed by check bitfield

Reply:
This proc shouldnt exist